src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
 - `js-beautify` for pretty generated HTML.
 
 - `json5` for more human readable representation of arbitrary values.
+
+---
+
+## Microdata and semantic tag usage (WIP)
+
+### Schema nodes to add microdata to
+* Article
+    * `div` `itemtype='schema:Article'`
+    * itemprops:
+        * authors* (author?): `ol`
+        * title*: `h1`? (+ `title`)
+        * environment: `data`?
+* Audio
+    * `audio`
+    * itemprops:
+        * caption*: `figcaption`
+        * transcript
+* Brand
+    * `picture`/`img` `itemtype='schema:Brand'`
+    * itemprops:
+        * name: maybe unnecessary, just use in alt tag?
+        * logo*: `source`/`image`
+        * reviews
+* Code
+    * `div` `itemtype='stencila:Code'`
+    * itemprops:
+        * language* (maybe important for setting syntax highlighting): `data`
+        * value*: `code`
+* CodeChunk
+    * `div` `itemtype='stencila:CodeChunk'`
+    * itemprops:
+        * outputs: `ol` of `samp`/`output` (possibly include `figure`/`table`)
+* CodeExpression
+    * `div` `itemtype='stencila:CodeExpr'`
+    * itemprops:
+        * value:`code` (syntax highlighting from Code node's language prop)
+* Collection
+    * `nav` `itemtype='schema:Collection'`
```

Comment:
I'm a little hesitant to make the assumption that all `Collections` are navigable collections. Using either `ol` or `ul` seems more natural to me, that way the user can decide when and where to wrap in a `nav` tag as needed